revolt/types/gateway.py

```diff
@@ -32,3 +34,16 @@ class ReadyEventPayload(BasePayload):
 
 class MessageEventPayload(BasePayload, Message):
     pass
+
+class MessageUpdateData(TypedDict):
+    content: str
+    edited: dict[str, Any]
```

Comment:
Is this able to be narrowed down even more so `Any` isn't needed? A typeddict here would be prefered

Reply:
yup, just fixed this sorry